Dockerfile

```diff
@@ -61,6 +61,8 @@ ENV OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
 
 # Other requirements and recommendations to run Odoo
 # See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
+RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main' >> /etc/apt/sources.list.d/postgresql.list
```

Comment:
You are adding the ubuntu version of the repos, please use Debian's if available.